packages/create-instantsearch-app/src/templates/InstantSearch.js/src/app.js.hbs

```diff
@@ -15,7 +15,7 @@ const search = instantsearch({
   indexName: '{{indexName}}',
   searchClient,
   future: { preserveSharedStateOnUnmount: true },
-  {{#if flags.insights}}insights: true,{{/if}}
+  {{#if enableInsights}}insights: true,{{/if}}
```

Comment:
We can transform the flag so this still is based on flags.insights, no? I think that would be more beautiful in template

Reply:
I think this changes the meaning from flags.insights = "supports the insights prop" to flags.insights = "enable insights", which is not great, even if it's internal.

I'd prefer leaving the `flags.insights` prop untouched for this reason.

Reply:
Oh okay, I see how the meaning is changed in `postProcessAnswers` for autocomplete. I'll copy this behaviour then!